### PR TITLE
NINA manifest

### PR DESCRIPTION
**Pull request by isbeorn**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/93755)

### CODE REVIEW

**Comment by isbeorn** [comment]
@microsoft-github-policy-service agree

**Comment by isbeorn** [comment]
> Hello @isbeorn,
> 
> One or more of the installer URLs doesn't appear valid.
> 
> This may happen for sites with policies prohibiting distribution or use by third parties.
> 
> This may happen for URLs pointing to domains that do not align with the publisher domain or package domain. If you could provide supporting evidence from the publisher that the URLs for the installer are correct, that would help us to validate and approve this PR.
> 
> Template: msftbot/validationError/urls/domain

The installer files are hosted by me on backblaze. How should i proceed to proof that this url is indeed valid?

File: manifests/s/StefanBerg/NINA/2.1.0.9001/StefanBerg.NINA.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+PackageIdentifier: StefanBerg.NINA
+PackageVersion: 2.1.0.9001
+PackageLocale: en-US
+Publisher: Stefan Berg
+PublisherUrl: https://www.patreon.com/stefanberg
+Author: Stefan Berg
+PackageName: N.I.N.A. Nighttime Imaging 'N' Astronomy
+PackageUrl: https://nighttime-imaging.eu/
```
Comment (stephengillie) [inline manifests/s/StefanBerg/NINA/2.1.0.9001/StefanBerg.NINA.locale.en-US.yaml:8]:
```suggestion
PackageUrl: https://nighttime-imaging.eu/download/
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**